check_strategies.py

```
@@ -0,0 +1,61 @@
+        "/api/v1/strategies/marketplace",
+        "/api/v1/strategies/owned",
+        "/api/v1/strategies/purchased",
+        "/api/v1/portfolio/strategies"
+    ]
+
+    for endpoint in endpoints:
+        url = f"https://cryptouniverse.onrender.com{endpoint}"
+        print(f"\n=== Testing {endpoint} ===")
+
+        # Try GET request
+        response = requests.get(url, headers=headers)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)